entity-service-impl/src/main/java/org/hypertrace/entity/query/service/EntityAttributeMapping.java

```diff
@@ -52,6 +67,10 @@
         .or(() -> this.calculateDocStorePathFromAttributeId(requestContext, attributeId));
   }
 
+  public List<String> getMultiValuedAttributes() {
+    return multiValueAttributes;
```

Comment:
Can we reverse this to something similar to above like `isMultiValued(RequestContext requestContext, String attributeId). The reason being this impl would only support v1-typed entities. V2 entities are not stored in the ES metadata, they're only available from attribute service (single source of truth) and we need to request them from there.

Reply:
Discussed a bit with Pavan. The `AttributeFilter` proto needs such a field for this to work in the current scheme of things.
Similar to how we convert the Attribute Id to the name field in AttributeFilter using the above method, we'll be doing the same for the isMultiValued field. Once the AttributeFilter has the isMultiValued field, the conversion to doc store query can be based on it.

Reply:
Why does the requester need to provide metadata? The attribute id should be all we need

Reply:
Can you please review and see if something can be changed ?

Reply:
I had suggested above - `boolean isMultiValued(RequestContext requestContext, String attributeId)` - did that run into issues? That's identical to what we're doing for getting the doc store path, so all the infrastructure is in place.

Reply:
I have added same.

Reply:
@aaron-steinfeld thats exactly what Pavan did. But, that alone is not enough because the converter which converts from AttributeFilter to doc store filter is where this info is needed. Which means it needs to be available in the AttributeFilter proto or is there a simpler way?

Reply:
Sorry, missed that there were new commits. Let me take a look